pages/common/license.md

```diff
@@ -0,0 +1,19 @@
+# license
+
+> Create license files for open-source projects.
+
+- Create a license:
```

Comment:
@rubenvereecken you're right, I @-mentioned the wrong person :) What I meant was that the next example, "create a license with **custom** filename" (emphasis mine) suggests that the vanilla one uses a default filename, so the assumption of output to stdout is shaken. If the description instead said something like "create a license and it save to a file", then we'd reinforce the assumption that the plain calling of the command outputs directly to stoud. It's not a big deal, of course, but we should strive to reduce confusion wherever possible.